l10n_ch_account_statement_base_import/parsers/ubs_file_parser.py

```diff
@@ -118,8 +118,8 @@ def _parse_transactions(self):
                 line.get("Description 3", '')]
             label = ' '.join(filter(None, descriptions))
 
-            debit = "D�bit".decode('iso-8859-15')
-            credit = "Cr�dit".decode('iso-8859-15')
+            debit = u"Débit".decode('iso-8859-15')
+            credit = u"Crédit".decode('iso-8859-15')

```

Comment:
Shouldn't it be `encode` instead of `decode` ? Source is unicode and you want `iso-8859-15`

This will raise 

    UnicodeEncodeError: 'ascii' codec can't encode character u'\xe9' in position 1: ordinal not in range(128)